Header header
geometry_msgs/Vector3 next_point
geometry_msgs/Twist next_vel
